### ISSUE TITLE
训练的迭代次数

### ISSUE THREAD

**Issue by Accept111** [huai-chang/DiffEIC#7]
作者你好！
我跑训练代码的时候，大概进行了十几万次迭代指标就差不多到了论文提到的数值。是一定要跑论文中提到的50万次迭代吗？
![image](https://github.com/user-attachments/assets/37aec39d-8783-4e8d-96b8-2414e089607a)

感谢作者的回复！

**Comment by huai-chang** [comment]
指标平稳后你就可以停止了，后续也不会有太大的提升了

**Comment by Accept111** [comment]
就是根据你训练这段代码的经验，大概迭代多少次就差不多了，我感觉不用五十万次。

**Comment by huai-chang** [comment]
就和论文里说的：``We first train the proposed DiffEIC with λ = 1 for 300K iterations, and then adapt them using target λ for another 200K iterations. We set the learning rate to 2×10−5 during the fine-tuning process.`` 我采用两阶段训练策略。为了保证性能达到最终的平稳，我把迭代次数设置的比较大。按照经验，第一阶段训练20w步其实就差不多了。第二阶段的话，差不多15w步吧

**Comment by Accept111** [comment]
1、意思就是我将λ设为1跑300k次迭代得到一个预训练模型，在这个模型的基础上然后我将λ设为我需要的值进行微调就可以了嘛？
2、如果我想修改一下你的网络结构，我是不是要从头开始训练？

**Comment by huai-chang** [comment]
1.是的 2.需要

**Comment by Accept111** [comment]
感谢作者回答
